Philips.CodeAnalysis.DuplicateCodeAnalyzer/AvoidDuplicateStringsAnalyzer.cs

```diff
@@ -0,0 +1,74 @@
+// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using System.Collections.Generic;
```

Comment:
I would keep this in MaintainabilityAnalyzers. DuplicateCodeAnalyzer is really a special case. (ie., read very slow, etc.) Folks opt into this separate package. Whereas they already have MaintainabilityAnalyzers and should get this one for "free".

Reply:
Moved it